src/common/yml/SfYamlParser.php

```diff
@@ -22,7 +22,7 @@
  * @author     Fabien Potencier <fabien.potencier@symfony-project.com>
  * @version    SVN: $Id: sfYamlParser.class.php 10832 2008-08-13 07:46:08Z fabien $
  */
-class sfYamlParser
+class SfYamlParser
```

Comment:
Yeah, this appears to be the old Symfony 1 yml component, let's not do too much maintenance on that :P

Reply:
This is indeed very old Symfony 1 code. It's been here since at least 2012, in a previous change set I must've changed the class names and it stopped working. I just changed them back. I plan on doing no further maintenance on anything to do with this component.

Reply:
I think it's best to not include the Yaml component in this commit, maybe a separate would be better